conans/client/cache/remote_registry.py

```diff
@@ -154,12 +170,27 @@ def _get_by_url(self, url):
 
     def rename(self, remote_name, new_remote_name):
         if new_remote_name in self._remotes:
-            raise ConanException("Remote '%s' already exists" % new_remote_name)
+            raise ConanException("Remote '%s' already exists" %
+                                 new_remote_name)
 
         remote = self._remotes[remote_name]
-        new_remote = Remote(new_remote_name, remote.url, remote.verify_ssl)
-        self._remotes = OrderedDict([(new_remote_name, new_remote) if k == remote_name
-                                     else (k, v) for k, v in self._remotes.items()])
+        new_remote = Remote(new_remote_name, remote.url, remote.verify_ssl,
+                            remote.disabled)
+        self._remotes = OrderedDict([
+            (new_remote_name, new_remote) if k == remote_name else (k, v)
+            for k, v in self._remotes.items()
+        ])
+
+    def set_disabled_state(self, remote_name, state):
+        filtered_remotes = []
+        for remote in self._remotes.values():
+            if fnmatch.fnmatch(remote.name, remote_name):
```

Comment:
I have one concern about `fnmatch`, what are the special characters allowed? can we have any trouble if a `remote_name` already contains one of them?

Maybe it is a good PR to forbid those special chars for remote names...

Reply:
https://docs.python.org/2/library/fnmatch.html

Not a big problem, it is possible to search for special characters using brackets, so knowing (docs/help) that it is doing a fnmatch comparison, should be enough.